plugins/arangodbendorsements/main.go

```diff
@@ -53,13 +52,6 @@ type ArangoEndorsementStore struct {
 	RelCollection  string
 }
 
-// PsaLinkObject represents an Edge Collection container in the DB
-type PsaLinkObject struct {
-	From     string `json:"_from"`
-	To       string `json:"_to"`
-	Relation string `json:"rel"`
-}
-
 //Resource describes the software information associated with a tag
```

Comment:
I realized after initial  commits, that it is a good golang convention to have a // <blank space> XXX, rather than //XXX
I am happy to do this post your commit in this section of the code and elsewhere!

Reply:
Any refactoring / cleanup is always welcome.  When you do, please raise an issue first.  This also needs to go together with an increase of test coverage to make sure we don't introduce bugs.